landoapi/uplift.py

```diff
@@ -145,8 +136,9 @@ def create_uplift_revision(
     )
 
     # Append an uplift mention to the summary
-    summary = phab.expect(source_revision, "fields", "summary")
-    summary += f"\nNOTE: Uplifted from D{source_revision['id']}"
+    summary = str(phab.expect(source_revision, "fields", "summary"))
+    summary += f"\nOriginal Revision: {phab.url_base}/D{source_revision['id']}"
+    summary = summary.strip()
```

Comment:
nit: is calling `str()` necessary here? Same for `summary.strip`. Perhaps we can strip the phab summary since the additional line is not contributing any extraneous blank space.

Reply:
I added the call to `str` because technically `phab.expect` doesn't return a concrete type, so my editor was complaining about type issues. I'm fine with removing it but it shouldn't change behaviour here, just give PyRight a hint that `summary` should be a `str`.

I added the call to `strip` to avoid having a blank line above `Original Revision` if the summary was empty before uplift. As I'm writing this I realize that we don't replace the `Differential Revision` line in the commit message in this iteration, so I'll need to change this anyways.